src/app/api/routes/route.ts

```
@@ -142,11 +143,75 @@ export async function GET(request: Request) {
       return NextResponse.json(error, { status: 404 });
     }
 
-    // Success response
+    // Access recommendations for journey beginning and end
+    let accessRecommendations;
+
+    // Only include recommendations if the distances are meaningful (> 50m)
+    const originRecommendation = fromStationCoordinates
+      ? getAccessRecommendation(
+          {
+            lat: parseFloat(fromStationCoordinates.split(',')[0]),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)